packages/adapters/database/src/client.ts

```diff
@@ -36,6 +36,7 @@ import { BigNumber } from "ethers";
 
 import { pool } from "./index";
 
+const DEFAULT_RECORDS_TO_STORE = 100;
```

Comment:
We should be able to store much more than 100 in the DB. Let's make this 1000 at least.

Reply:
https://github.com/connext/monorepo/pull/4497/commits/b2a5a02f0e1c74f31cfb285066d4138ca7c9199d